core/src/main/scala/org/bitcoins/core/protocol/ln/LnTagPrefix.scala

```diff
@@ -19,7 +19,7 @@ object LnTagPrefix {
     override def value: String = "d"
   }
 
-  case object SignaturePubKey extends LnTagPrefix {
+  case object ScriptPubKey extends LnTagPrefix {
```

Comment:
Again, this is for tag `n` which denotes the "33-byte public key of the payee node". The name SignaturePubKey probably comes from this line in BOLT-11: "A writer MAY include one n field, which MUST be set to the public key used to create the signature." We could probably rename to something more intuitive?